test/integration/forms/action.js

```diff
@@ -325,10 +321,8 @@ context('Patient Action Form', function() {
   });
 
   specify('discarding stored submission', function() {
```

Comment:
Same pattern could be used in the `Patient Form` => `discarding stored submission` test as well? Looks like that one is still using the `currentTs` method from before.

Code link: https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/integration/forms/patient.js#L309

Reply:
Well nvm, I guess that one is just doing `const currentTs = dayjs();`... so maybe that one won't cause issues...

Reply:
yeah.. should probably be updated for consistency